source/sass/style.scss

```
@@ -23,3 +22,4 @@
 @import "./blocks/radio.scss";
 @import "./blocks/form-options.scss";
 @import "./blocks/checkbox.scss";
+@import "global.scss";
```

А почему без точки, путь то в одном стиле стоит писать )
```suggestion
@import "./global.scss";
```